README.md

```diff
@@ -29,6 +29,7 @@ $ npm install jsonwebtoken
 
 `secretOrPrivateKey` is a string, buffer, or object containing either the secret for HMAC algorithms or the PEM
 encoded private key for RSA and ECDSA. In case of a private key with passphrase an object `{ key, passphrase }` can be used (based on [crypto documentation](https://nodejs.org/api/crypto.html#crypto_sign_sign_private_key_output_format)), in this case be sure you pass the `algorithm` option.
+If `jwt.verify` is called asynchronous, `secretOrPublicKey` can be a function that should fetch the secret or public key. See below for a detailed example
```

Comment:
This is in wrong place, right? it should be under `jwt.verify` section, where we should indicate `secretOrPublicKey` can be a function too.